### DISCUSSION TITLE
Call for Article Proposals for Open Source Expanded Column in IEEE Computer Magazine

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#52] [3️⃣ 聊学术]
Dirk 教授在他的主页上发布了一则征稿信息，我们对我们来说可以考虑投一个。

https://dirkriehle.com/2022/05/23/call-for-article-proposals-for-open-source-expanded-column-in-ieee-computer-magazine/

![image](https://user-images.githubusercontent.com/15010826/170592369-58a063a4-0ab4-44c0-9942-9f2e10b59f6a.png)

可能的方向包括：
- OpenDigger
- OSPO in China (with Alibaba)
- GitHub insight 报告
- 开源价值网络

大家看看有啥好主意~

**Comment by bifenglin** [comment]
这个思维导图确实从一些方向给出了很不错的建议。

**Reply by will-ww** [reply]
> 这个思维导图确实从一些方向给出了很不错的建议。

这个就是 Dirk 教授的分类体系~ :-)

**Comment by xiaoya-yaya** [comment]
报名🙋🏻‍♀️

**Reply by will-ww** [reply]
好呀，你来做个一作好了～

**Reply by xiaoya-yaya** [reply]
我可以最近整理一下 proposal 先邮件给 Dirk 教授～我觉得 OSPO 这个话题就很不错，正好可以把和阿里在 OSSNA 上的演讲沉淀下来。个人感觉有两个方向可以考虑，看是要写大一点还是写小一点：
- 往大了写，可以做一个中国 OSPO 现状（成熟度，Business Model 等）研究；
- 往小了写，可以就以阿里作为一个 case study 探讨中国企业的 OSPO 的发展与模式。

**Reply by wangleo61** [reply]
宝藏宝藏，加入哈

**Reply by will-ww** [reply]
欢迎，一起～

**Comment by wangleo61** [comment]
太荣幸啦


发件人: Will ***@***.***>
发送时间: Thursday, June 2, 2022 4:10 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [X-lab2017/open-wonderland] Call for Article Proposals for Open Source Expanded Column in IEEE Computer Magazine (Discussion #52)


欢迎，一起～

―
